src/applications/vaos/reducers/expressCare.js

```diff
@@ -92,54 +87,30 @@ export default function expressCareReducer(state = initialState, action) {
         windowsStatus: FETCH_STATUS.loading,
       };
     case FETCH_EXPRESS_CARE_WINDOWS_SUCCEEDED: {
-      const { facilityData, nowUtc } = action;
-      const windows = []
-        .concat(...facilityData)
-        .filter(f => !!f.expressTimes)
-        .map(({ expressTimes, authoritativeName, rootStationCode, id }) => {
-          const { start, end, offsetUtc, timezone } = expressTimes;
-          const today = nowUtc.format('YYYY-MM-DD');
```

Comment:
I think this code that I wrote is probably what was causing the failing tests after a certain time.  Should have been using the facility offset to figure out `today`, because after a certain time, UTC date is ahead of most US time zones.

Reply:
Yeah, could be, I made the same mistake in the new code and had to track that down this morning.